test/go.mod

```diff
@@ -5,21 +5,19 @@ go 1.20
 require (
 	github.com/metal3-io/baremetal-operator/apis v0.4.0
 	github.com/metal3-io/cluster-api-provider-metal3/test v1.5.1
-	github.com/onsi/ginkgo/v2 v2.12.0
-	github.com/onsi/gomega v1.27.10
+	github.com/onsi/ginkgo/v2 v2.13.0
```

Comment:
The ginkgo version is unfortunately also defined in the Makefile: https://github.com/metal3-io/baremetal-operator/blob/dfc23753f8c37dc565e2e487a18c65845f8169e2/Makefile#L25
Can you update that as well?

Reply:
Done, good catch. Is it there for the e2e?

Reply:
Yeah I added it together with the e2e tests. Probably we could do it in a better way so that we don't need the separate version there